include/bluetooth/conn.h

```diff
@@ -267,6 +356,27 @@ int bt_conn_get_remote_info(struct bt_conn *conn,
 int bt_conn_le_param_update(struct bt_conn *conn,
 			    const struct bt_le_conn_param *param);
 
+/** @brief Update the connection transmit data length parameters.
+ *
+ *  @param conn  Connection object.
+ *  @param param Updated data length parameters, only transmit parameters will
+ *               be used.
```

Comment:
No need to mention transmit parameters, as those are the only ones present in `struct bt_conn_le_data_len_param`
```suggestion
 *  @param param Updated data length parameters.
```

Reply:
Good catch. The RX parameters was here earlier.